### PR TITLE
New EasyConfig for NetLogo

### PR DESCRIPTION
**Pull request by valtandor**

### CODE REVIEW

File: easybuild/easyconfigs/n/NetLogo/NetLogo-5.3.1-64.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# NetLogo is precompiled and needs no particular toolchain.
+# It even comes with its own Java.
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://ccl.northwestern.edu/netlogo/%(version)s/']
+sources = ['%(name)s-%(version)s%(versionsuffix)s.tgz']
+
+sanity_check_paths = {
+    'files': ['NetLogo', 'NetLogo3D', 'NetLogoLogging'],
```
Comment (boegel) [inline easybuild/easyconfigs/n/NetLogo/NetLogo-5.3.1-64.eb]: @valtandor should we update `$PATH` (via `modextravars`) for these?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/076810103d70a7821692fa7cbc454d09 for a full test report.

File: easybuild/easyconfigs/n/NetLogo/NetLogo-5.3.1-64.eb (hunk 1 above)
Comment (valtandor) [inline easybuild/easyconfigs/n/NetLogo/NetLogo-5.3.1-64.eb]: Probably not a bad idea, but I'd ordinarily use `prepend-path` (in the module file) for that purpose. Can `modextravars` be used to create a `prepend-path` statement?

Comment (boegel) [inline easybuild/easyconfigs/n/NetLogo/NetLogo-5.3.1-64.eb]: good point, you should actually be using `modextrapaths`, which does indeed using `prepend-path` statements

**Comment by boegel** [comment]
@valtandor ah, no, indeed, this is taken care of by the `Binary` easyblock (`PackedBinary` derives from that), so this looks good to go

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2301.phanpy.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fe1e5dd4ce23abd02a2e70496ed2a9af for a full test report.

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**